apps/stress-test/src/shared/react/TestMount.tsx

```diff
@@ -18,7 +18,7 @@ const debouncedOnRender: DebouncedOnRender = () => {
       clearTimeout(timeoutId);
     }
 
-    timeoutId = setTimeout(() => {
+    timeoutId = (setTimeout(() => {
```

Comment:
using globals is antipattern and root of issues not just on type level. use:

```suggestion
    timeoutId = (window.setTimeout(() => {
```